projects/Mallard/src/screens/home-screen.tsx

```diff
@@ -248,15 +247,14 @@ const IssueListView = withNavigation(
 			const listRef = useRef<FlatList<IssueSummary>>();
 			const prevCurrentIndexRef = useRef<number>(currentIssueIndex);
 			useEffect(() => {
-				if (listRef.current == null || currentIssueIndex < 0) {
+				if (listRef.current === undefined || currentIssueIndex < 0) {
```

Comment:
I don't like changing `null` to `undefined` but the linter says there's no crossover between `listref.current` and `null` and i believe it